### DISCUSSION TITLE
[PtBr] Brazilian Portuguese Language File

### DISCUSSION THREAD

**Discussion by WanCaSil** [aclap-dev/video-downloadhelper#990] [Language & Translation]
[pt_BR.txt](https://github.com/user-attachments/files/16160104/pt_BR.txt)

**Comment by jedelucca** [comment]
5 strings que estavam faltando

[pt_BR.txt](https://github.com/user-attachments/files/16186459/pt_BR.txt)

**Comment by Agronopolos** [comment]
[pt_BR.txt](https://github.com/user-attachments/files/16242258/pt_BR.txt)
Hi peoples... 
Complete translation

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
